build-id-versioning/starter/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"github.com/pborman/uuid"
+
+	"go.temporal.io/sdk/client"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Get task queue name from CLI arg
+	taskQueue := os.Args[1]
```

Comment:
Most of our samples just use a hardcoded task queue and don't require CLI input, I think this can do that too

Reply:
It needs to start with an unversioned queue every time to work. The alternative would be moving the starter and workers into one file.

Reply:
I think `"build-id-versioning-" + uuid.New()` in the worker can just be a fixed value too

Reply:
That means re-running the sample breaks - it needs to be a new TQ each time.

Reply:
> That means re-running the sample breaks

This is the case with many of our samples